README.md

```diff
@@ -51,6 +54,8 @@ This project is licensed under the terms of the MIT license. See [LICENSE](LICEN
 
 ## Changelog
 
+* v 1.0.2
+  * Fix for "Insert -> Derive" missing the insert panel. Issue #1.
```

Comment:
Added update to changelog for you to make things super easy, I tried to keep the same format you use